packages/web/lib/logging/server.ts

```
@@ -2,30 +2,40 @@ import winston from 'winston'
 import DailyRotateFile from 'winston-daily-rotate-file'
 import * as Sentry from '@sentry/nextjs'
 import { LogLevel, LogMetadata, formatError } from './index'
+import { Transform } from 'stream'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)